apps/partner2/stylesheets/overview.styl

```diff
@@ -20,15 +20,19 @@
       flex-basis calc(66% \- 40px)
       margin-right 40px
     &[data-module=news]
-      flex-basis 33%
+      .partner2-news
+        display flex
+      .partner2-news-item
+        flex-basis 33%
```

Comment:
My initial reaction is how come it's not 33.33 as below:) But that does not really matter much.